processing/dwf_calculation_algorithm.py

```diff
@@ -0,0 +1,396 @@
+# -*- coding: utf-8 -*-
+
+"""
+/***************************************************************************
+ DWFCalculator
+                                 A QGIS plugin
+ Calculate DWF
+ Generated by Plugin Builder: http://g-sherman.github.io/Qgis-Plugin-Builder/
+                              -------------------
+        begin                : 2021-01-27
+        copyright            : (C) 2021 by Nelen en Schuurmans
+        email                : emile.debadts@nelen-schuurmans.nl
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+"""
+
+__author__ = 'Nelen en Schuurmans'
+__date__ = '2021-01-27'
+__copyright__ = '(C) 2021 by Nelen en Schuurmans'
+
+# This will get replaced with a git SHA1 when you do a git archive
+
+__revision__ = '$Format:%H$'
+
+
+import os
+import sqlite3
+import json
+import csv
+from datetime import datetime
+
+from qgis.PyQt.QtCore import QCoreApplication
+from qgis.core import (QgsProcessing,
+                       QgsFeatureSink,
+                       QgsProcessingAlgorithm,
+                       QgsProcessingParameterFeatureSource,
+                       QgsProcessingParameterFeatureSink,
+                       QgsProcessingParameterFile,
+                       QgsProcessingParameterString,
+                       QgsProcessingParameterFileDestination,
+                       QgsProcessingParameterProviderConnection,
+                       QgsProviderRegistry,
+                       QgsProviderConnectionException,
+                       QgsProcessingException)
```

Comment:
This way of importing doesn't match the style in the rest of the code: one full import per line.
Running "make beautiful" will solve it, though. I'll do that.